src/test/kotlin/com/terraformation/backend/support/atlassian/AtlassianHttpClientExternalTest.kt

```diff
@@ -50,29 +51,45 @@ class AtlassianHttpClientExternalTest : RunsAsUser {
                     apiClientUsernamePrefix = "test"))
 
     client = AtlassianHttpClient(config)
-    requestTypes = client.requestTypes
 
+    requestTypes = client.requestTypes
     assertTrue(requestTypes.isNotEmpty())
   }
 
   @AfterEach
   fun deleteCreatedIssues() {
-    createdIssueIds.forEach { client.deleteIssue(it) }
-    createdIssueIds.clear()
+    //        createdIssueIds.forEach { client.deleteIssue(it) }
+    //        createdIssueIds.clear()
```

Comment:
Will this leave test issues behind in Jira?

Reply:
Oops yes I meant to uncomment that. I forgot to do so because I wanted to manually check the issue. Let me uncomment it. At some point maybe this can be a test config.